inference-engine/tests/functional/inference_engine/ir_serialization/serialize.cpp

```diff
@@ -46,7 +46,24 @@ TEST_F(SerializationTest, BasicModel) {
     ASSERT_TRUE(success) << message;
 }
 
-TEST_F(SerializationTest, ModelWithMultipleOutputs) {
+// DISABLED: C++ exception with description "Invalid IR! Y name is not unique!
+TEST_F(SerializationTest, DISABLED_BasicModel_ONNXImporter) {
```

Comment:
@ilyachur I added tests with ONNX Importer path. The same model which pases on MO path is failing on ONNX Importer path - model is serialized but it cannot be read back because output layer don't have unique names. Is it known issue ? Should I open new bug for it ?

Reply:
Yes it is a known issue, that all names in the CNNNetwork should be unique.
But why cannot you create result layers with unique names? For example you can generate the name using the next algorithm: [name of previous op] + "." + [index of output port]

Reply:
Yeah I known before that names in CNNNetwork should be unique and I **assumed** it is handled when ng function is created and at the point of serialization names returned via node->get_friendly_name() are unique. So, I'm using node->get friendly_name() to retrieve name for layer and I'm not creating it myself. Is my assumption is incorrect?

So, the bug I'm talking about is that ONNX Importer is not creating unique names.  Adding name logic creation yet in another place seems like not clean solution. In my mind it should be handled at point of graph creation. Looks like it is handled correctly in MO path but not in ONNX importer path.

Reply:
We talked offline and agreed that I'll provide WA in serialization code which will rename layer if it's name is not unique in ng function.